stdlib/Distributed/test/distributed_exec.jl

```diff
@@ -15,6 +15,11 @@ include(joinpath(Sys.BINDIR, "..", "share", "julia", "test", "testenv.jl"))
         1
     end
 
+# PR #28651
```

Comment:
Maybe
```julia
for T in (Int64, Int128)
  n = @distributed (+) for i in Base.OneTo(T(10))
    i
  end
  @test n == 55
end
```
This only happens whenever `length` returns something that is not an `Int` which is relatively rare.

Reply:
Yep. I had something written out last night but yours is simpler. Thanks.